libraries/rtos/rtx/TARGET_CORTEX_M/RTX_Conf_CM.c

```diff
@@ -357,18 +357,17 @@ void os_idle_demon (void) {
 extern void error(const char* format, ...);
 extern osThreadId svcThreadGetId (void);
 
-#include "stdio.h"
-
 void os_error (uint32_t err_code) {
     /* This function is called when a runtime error is detected. Parameter     */
     /* 'err_code' holds the runtime error code (defined in RTX_Config.h).      */
     osThreadId err_task = svcThreadGetId();
-    error("CMSIS RTX error code: 0x%08X, task ID: 0x%08X\n", err_code, (uint32_t)err_task);
```

Comment:
NOTE: `err_task` doesn't need an explicit cast, since it will be automatically promoted during the variadic function call.